src/medVtkInria/vtkDataManagement/vtkMetaDataSet.cxx

```diff
@@ -912,6 +912,39 @@ vtkDataArray* vtkMetaDataSet::GetArray (const char* name)
   return ret;
 }
 
+void vtkMetaDataSet::ClearInputStream(std::ifstream& file)
+{
+  file.clear();
+  file.seekg(0, file.beg);
+}
+
+bool vtkMetaDataSet::PlaceStreamCursor(std::ifstream& file, const char* token)
+{
+  char buf[256];
+  file >> buf;
+
```

Comment:
The code below works if the headers are separated by whitespace. Is this true ? In any case it should probably be renamed to something like ```PlaceStreamCursorOnHeader``` because the current name suggests it allows searching for any token (whitespace or not).

 Also, is there any way to know that we've read the last header ? When reading a non-medit type file, this function will search through the whole file which may be unnecessarily long.

Reply:
I use this method to find any token in files. It is true however that when reading non-medit files the whole file will be read for nothing.
I'll try to find a more efficient way to quickly find the header.